### PR TITLE
akolosov: add functions print_max and return_max

### PR DESCRIPTION
**Pull request by kazh000**
Lession2: tasks 1 and 2

### CODE REVIEW

File: Practice/akolosov/print_max.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    else:
+        print(f"Max value: {b}")
+
+
+# How many ways can I call the function?
+print_max(1, 11)
+
+print_max(b=2, a=55)
+
+my_list=[3, 99]
```
Comment (IlyaOrlov) [inline Practice/akolosov/print_max.py:13]: В строках 13, 16 и 19 не хватает пробелов вокруг "=".
См. pep8: "Always surround these binary operators with a single space on either side: assignment (=), augmented assignment (+=, -= etc.), comparisons (==, <, >, !=, <>, <=, >=, in, not in, is, is not), Booleans (and, or, not)." (https://www.python.org/dev/peps/pep-0008/#:~:text=Always%20surround%20these%20binary%20operators%20with%20a%20single%20space%20on%20either%20side%3A%20assignment%20(%3D)%2C%20augmented%20assignment%20(%2B%3D%2C%20-%3D%20etc.)%2C%20comparisons%20(%3D%3D%2C%20%3C%2C%20%3E%2C%20!%3D%2C%20%3C%3E%2C%20%3C%3D%2C%20%3E%3D%2C%20in%2C%20not%20in%2C%20is%2C%20is%20not)%2C%20Booleans%20(and%2C%20or%2C%20not).)

File: Practice/akolosov/return_max.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+def return_max(a: int, b: int):
+    if a > b:
+        res = a
+    else:
+        res = b
+    return res
+
+
+a=9
```
Comment (IlyaOrlov) [inline Practice/akolosov/return_max.py:9]: В строках 9 и 10 не хватает пробелов вокруг "=".

**Review by IlyaOrlov — CHANGES_REQUESTED** [review]

File: Practice/akolosov/print_max.py (hunk 1 above)
Comment (kazh000) [reply Practice/akolosov/print_max.py]: Привычка с других языков :(. Поправил.

File: Practice/akolosov/return_max.py (hunk 2 above)
Comment (kazh000) [reply Practice/akolosov/return_max.py]: Тоже поправил

**Review by IlyaOrlov — APPROVED** [review]

**Pull request merged**